package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "step236-2020",
```

Comment:
we can have a more expressive name here.

Reply:
Replaced with scriba. Do we need more descriptive, or only title is ok?